docs/guide/debugging-teal.md

```diff
@@ -16,7 +16,7 @@ If it displays the `tealdbg` help, you're good to go.
 
 ## Using TEAL debugger with algob
 
-Creating transaction data (via `goal --dryrun-dump` or SDK) could be a lengthy process, especially when using a transaction group. `Algob` provides an easy way to use debugger: by simply supplying the transactions as an input to the `TealDbg` method (same transaction parameters that we supply to [executeTx](https://algobuilder.dev/api/algob/modules.html#executetransaction) to execute same transaction on network).
+Creating transaction data (via `goal --dryrun-dump` or SDK) could be a lengthy process, especially when using a transaction group. `Algob` provides an easy way to use debugger: by simply supplying the transactions as an input to the `TealDbg` method (same transaction parameters that we supply to [executeTx](https://algobuilder.dev/api/algob/modules.html#executeTransaction) to execute same transaction on network).
```

Comment:
We renamed executeTransaction to executeTx. Let create TODO for it. We can update link after released new version.